deepspeech/results.txt

```diff
@@ -1 +1 @@
-nine months old
+nine months old
```

Comment:
I think it's complaining about this file for some reason?

Reply:
I have no clue how github works lmao just trying to do my reviewer duty

Reply:
awe man im not sure what the dif is

Reply:
@camwr i think it is just a file generated by deepspeech so i deleted